src/db_ecbs.cpp

```diff
@@ -96,25 +95,8 @@ int main(int argc, char* argv[]) {
 
     // load problem description
     YAML::Node env = YAML::LoadFile(inputFile);
-    std::vector<fcl::CollisionObjectf *> obstacles;
     std::vector<std::vector<fcl::Vector3f>> positions;
     std::vector<std::shared_ptr<fcl::CollisionGeometryd>> collision_geometries;
-    for (const auto &obs : env["environment"]["obstacles"])
-    {
-        if (obs["type"].as<std::string>() == "box"){
-            const auto &size = obs["size"];
-            std::shared_ptr<fcl::CollisionGeometryf> geom;
-            geom.reset(new fcl::Boxf(size[0].as<float>(), size[1].as<float>(), 1.0));
-            const auto &center = obs["center"];
-            auto co = new fcl::CollisionObjectf(geom);
-            co->setTranslation(fcl::Vector3f(center[0].as<float>(), center[1].as<float>(), 0));
-            co->computeAABB();
-            obstacles.push_back(co);
-        }
-        else {
-        throw std::runtime_error("Unknown obstacle type!");
-        }
-    }
```

Comment:
why does this need to be removed? Octomap support should be in addition?

Reply:
no, it was a mistake in db-cbs. Now we do it while creating a Problem object. You can look at dynoplan script.